docs/columns/vernacular_speech_language.md

```diff
@@ -1,6 +1,7 @@
 {% docs vernacular_speech_language %}
 
-Specifies the language of the vernacular speech. Extracts "mandarin", "malay", or "tamil" if the speech is vernacular, otherwise NULL.
+Specifies the language of the vernacular speech. 
```

Comment:
suggestion: Start sentence with "language" (which can be considered a "language" datatype by means of an enum)